script.js

```diff
@@ -4,17 +4,25 @@ const sendButton = document.getElementById('send');
 const clearButton = document.getElementById('clear');
 
 
-function display(){
+
+//This function will display the message at the bottom of the page in the <p> element
+sendButton.addEventListener('click', function() {
```

Comment:
Add `event` as an argument to the anonymous function, like this:
```
sendButton.addEventListener('click', function(event) {
```

Then inspect the `event` in the function for the data you need (e.g. (`console.log(event);` for starters).